configure.ac

```diff
@@ -463,37 +497,55 @@ fi
 test "${enable_dbengine}" = "yes" -a -z "${JUDY_LIBS}" && \
     AC_MSG_ERROR([libJudy required but not found. Try installing 'libjudy-dev' or 'Judy-devel'.])
 
-test "${enable_https}" = "yes" -a -z "${SSL_LIBS}" && \
-    AC_MSG_ERROR([OpenSSL required for HTTPS but not found. Try installing 'libssl-dev' or 'openssl-devel'.])
+test "${enable_https}" = "yes" -a -z "${SSL_LIBS}" -a -z  "${WOLFSSL_LIBS}" && \
+    AC_MSG_ERROR([OpenSSL or WolfSSL are required for HTTPS but not found. Try installing 'libssl-dev', 'openssl-devel' or 'libwolfssl-dev'.])
 
 test "${enable_dbengine}" = "yes" -a -z "${SSL_LIBS}" && \
```

Comment:
You added `-a -z  "${WOLFSSL_LIBS}"` in the previous test. Shouldn't you add it here as well?